sql-plugin/src/main/spark330db/scala/com/nvidia/spark/rapids/DatabricksShimServiceProvider.scala

```diff
@@ -24,7 +24,12 @@ package com.nvidia.spark.rapids
 
 object DatabricksShimServiceProvider {
   val log = org.slf4j.LoggerFactory.getLogger(getClass().getName().stripSuffix("$"))
-  def matchesVersion(dbrVersion: String): Boolean = {
+
+  def matchesVersion(dbrVersion: String,
+    shimMatchEnabled: Boolean = true,
```

Comment:
nit: I would just keep this the same name as you have in the SparkShimProvider i.e. `shimEnabled` or `isShimEnabled`